force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomLabels xmlns="http://soap.sforce.com/2006/04/metadata">
+    <labels>
+        <fullName>DatabaseService_NoInsertFlsException</fullName>
+        <categories>DatabaseService,Exception</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>The user does not have create access to fields required for this process.</shortDescription>
+        <value>You don&apos;t have field-level access to create this record. Contact your system administrator for help.</value>
+    </labels>  
+    <labels>
+        <fullName>DatabaseService_NoUpdateFlsException</fullName>
+        <categories>DatabaseService,Exception</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>The user does not have edit or update access to required fields.</shortDescription>
+        <value>You don&apos;t have field-level access to edit this record. Contact your system administrator for help.</value>
+    </labels>
+    <labels>
+        <fullName>DatabaseService_NoDeleteAccessException</fullName>
+        <categories>DatabaseService,Exception</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>The user does not have delete access to the object.</shortDescription>
+        <value>You don&apos;t have delete access to this object. Ask your Salesforce admin for help.</value>
+    </labels>
+</CustomLabels>
```

Comment:
@nicolersmith , I copied these Custom Labels for exception messages from GMS:
- `DatabaseService_NoInsertFlsException` is copied from [Does_Not_Have_Create_FLS_Access_Message](https://github.com/SalesforceFoundation/Grants-Management/blob/feature/230/force-app/main/default/labels/CustomLabels.labels-meta.xml#L459-L466)
- `DatabaseService_NoUpdateFlsException ` is copied from [Does_Not_Have_Edit_FLS_Access_Message](https://github.com/SalesforceFoundation/Grants-Management/blob/feature/230/force-app/main/default/labels/CustomLabels.labels-meta.xml#L467-L474)
- `DatabaseService_NoDeleteAccessException ` is copied from [Does_Not_Have_Delete_Access_For_Object_Message](https://github.com/SalesforceFoundation/Grants-Management/blob/feature/230/force-app/main/default/labels/CustomLabels.labels-meta.xml#L443-L450)

Reply:
Can you update the second sentences to read "Ask your Salesforce admin for help." instead of "Contact your system administrator for help"?

Reply:
Yes

Reply:
Pushed

Reply:
Sorry, Scott. The second sentences for DatabaseService_NoInsertFlsException and DatabaseService_NoUpdateFlsException should be: "Ask your Salesforce admin for help."

Reply:
I got it backwards.  Fixed and pushed.